Readme.md

```diff
@@ -24,14 +24,14 @@ You find working examples for different use-cases under the [examples](./example
 
 ## Support Matrix
 
-| Component | Consume | Publish | CloudEvents | Metrics | Compression | Custom |
-| --- | --- | --- | --- | --- | --- | --- |
-| RabbitMQ | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | priority, dynamic routing |
-| Kafka | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | partitioning key, dynamic topic |
-| Http | (:heavy_check_mark:) | (:heavy_check_mark:) | :x: |:heavy_check_mark: | :x: | |
-| Timer | (:heavy_check_mark:) | - | :x: | :x: | :x: | |
-| SQS | (:heavy_check_mark:) | - | :x: | :x: | :x: | |
-| NATS | (:heavy_check_mark:) | (:heavy_check_mark:) | :x: | :x: | :x: | |
+| Component | Consume              | Publish              | CloudEvents (Protocol) | CloudEvents (JSON) | Metrics            | Compression        | Custom                          |
```

Comment:
Same here, the matrix was mostly self explanatory before, but now it's confusing to have CloudEvents two times in here and I'm wondering why JSON is named explicitly, but not for example Protobuf. So, is Protobuf now supported any more or what does this mean?

Reply:
DIfference between CloudEvent format json or protocol. Not protobuf

Reply:
And CloudEvent format Protocol means that the CloudEvent will be published just like before, whereas CloudEvent.Json serializes the CloudEvent to JSON with a specific, standardized format, correct?
Protobuf is then still supported by using CloudEvent.Protocol together with a Proto serializer, right?

Then I don't really understand the need to add `CloudEvent.Json` to this table. It just sounds to me like we have improved the way we serialize CloudEvents to JSON by confirming to a standard.

Maybe we could instead add another section to the Readme about _CloudEvent conformity_ where we mention that our JSON format confirms with the CloudEvent JSON spec, but Protobuf does not.

Reply:
Protobuf if still supported by both options as stated previously: either as `data_base64` (JSON) or as binary payload of the message itself.
The difference is that there are CloudEvent specs for Kafka, AMQP, ... and a general JSON event standard that can be used protocol agnostic.

Previously we were tight to the protocol specific CloudEvent specifications to encode and transport metadata and now we also support the protocol independent JSON format.

I think that fits rather good to the table but adding another section about CloudEvents and there handling could be helpful indeed. Is that something that'd comfort you?